pallets/did/src/utils.rs

```diff
@@ -16,17 +16,45 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
+use crate::{
+	did_details::{DidCreationDetails, DidPublicKey},
+	BalanceOf, Config, KeyIdOf,
+};
 use parity_scale_codec::Encode;
-use sp_runtime::traits::Hash;
+use sp_core::Get;
+use sp_runtime::traits::{Hash, Zero};
 use sp_std::vec::Vec;
 
-use crate::{did_details::DidPublicKey, Config, KeyIdOf};
-
 pub fn calculate_key_id<T: Config>(key: &DidPublicKey) -> KeyIdOf<T> {
 	let hashed_values: Vec<u8> = key.encode();
 	T::Hashing::hash(&hashed_values)
 }
 
+pub fn calculate_deposit<T: Config>(details: &DidCreationDetails<T>) -> BalanceOf<T>
+where
+	BalanceOf<T>: From<u32>,
+{
+	let mut deposit: BalanceOf<T> = T::BaseDeposit::get();
+
+	let count_service_endpoint: BalanceOf<T> = (details.new_service_details.len() as u32).into();
```

Comment:
```suggestion
	let service_endpoint_count: BalanceOf<T> = (details.new_service_details.len() as u32).into();
```

Reply:
We should consider this lint: https://rust-lang.github.io/rust-clippy/rust-1.68.0/index.html#as_conversions

Reply:
Or this: https://rust-lang.github.io/rust-clippy/rust-1.68.0/index.html#cast_possible_wrap